docs/README.md

```diff
@@ -1622,7 +1623,7 @@ export class GetProductsCount {
 
 ### Authentication and Authorization
 
-You can use the [Authentication Rocket](https://github.com/boostercloud/rocket-auth-aws-infrastructure) for adding the authentication and authorization to you application. But first, you need to know that authorization in Booster is done through roles. Every Command and ReadModel has an `authorize` policy that tells Booster who can execute or access it. It consists of one of the following two values:
+First of all, you need to know that the authorization in Booster is done through roles. Every Command and ReadModel has an `authorize` policy that tells Booster who can execute or access it. It consists of one of the following two values:
```

Comment:
I would add here that to use auth, the need to install the rocket, So it's clear since the beginning. Maybe something like _" To enable authorization in Booster, you have to install the authorization rocket which is compatible with JWT thrid party auth providers. In this section, we will cover how to install and use this rocket"_